app/views/includes/FurloughCapHelper.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.includes
+
+import java.time.Month
+import java.time.format.TextStyle
+import java.util.Locale
+
+import com.google.inject.Inject
+import models.{FullPeriodCap, FurloughCap, PartialPeriodCap, PeriodSpansMonthCap}
+import play.api.i18n.Messages
+
+class FurloughCapHelper @Inject()() {
+
+  def calculationFor(cap: FurloughCap)(implicit messages: Messages): Option[String] =
+    cap match {
+      case FullPeriodCap(_) => None
```

Comment:
Is this a potentially partial match? `(@annotation.switch cap match {` should issue compiler warnings or errors when we miss stuff out.

Reply:
I actually think this function will be backed out as the prototype has since changed to just link away to govuk guidance for 'How was the furlough cap calculated', so I wouldn't worry too much about what is in here as it will probably be removed before release